src/main/java/redis/clients/jedis/params/CopyParams.java

```diff
@@ -0,0 +1,47 @@
+package redis.clients.jedis.params;
+
+import static redis.clients.jedis.Protocol.Keyword.DB;
+import static redis.clients.jedis.Protocol.Keyword.REPLACE;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import redis.clients.jedis.Protocol;
+
+public class CopyParams extends Params {
+
+  private Long destinationDb;
+
+  private boolean replace;
+
+  public static CopyParams copyParams() {
+    return new CopyParams();
+  }
+
+  public CopyParams destinationDb(long destinationDb) {
```

Comment:
`db` can't be used in cluster mode. But `CopyParams` is directly used in cluster classes. This is contradictory.

Reply:
em. Do you have any suggestions?